doc/faq.md

```diff
@@ -222,9 +222,9 @@ address of 10.61.0.25, and VLAN102 being used for client traffic.
 
 #### Things to note
 
-* eth0 is the Management interface, with the default gateway.
-* vlan102 uses eth1.
-* br102 uses vlan102, and _has a bogus /32 IP address assigned to it_
+* `eth0` is the Management interface, with the default gateway.
+* `vlan102` uses `eth1`.
+* `br102` uses `vlan102`, and _has a bogus /32 IP address assigned to it_
```

Comment:
What is the "_" next to "has" for? Should we keep it?

Reply:
It renders in italics, but I agree - we should get rid of it.
That FAQ desperately needs a cleanup anyway. ;)